### DISCUSSION TITLE
How are repodata.json updates triggered?

### DISCUSSION THREAD

**Discussion by m3vaz** [conda/infra#664] [Q&A] [answered]
What triggers repodata.json updates?

12.0 `cuda` packages have been live for 3 hours and 12.0 `cuda-python` packages have been live for 6 hours on the `nvidia` channel. But neither is present in the repodata.json and are not coming up on `conda search`

**Comment by m3vaz** [comment]
Packages still not available when just searching the channel, but if you search `-c nvidia/label/main` it resolves properly

**Comment by barabo** [comment]
We believe this is related to a recent release of `conda` and are looking into it.

**Comment by jakirkham** [comment]
cc @quasiben @raydouglass

**Comment by jezdez** [comment]
To quote [a related issue in the conda repo](https://github.com/conda/conda/issues/12163#issuecomment-1344776643):

> We discovered an issue in our channel clone for nvidia that seems it could be related to the recent release of conda. I've bypassed the CDN for nvidia's channel for now, though, so it should be operational - although it may be slower than normal.
>
> Follow along in the statuspage for more details: https://anaconda.statuspage.io/incidents/zlywfxhfw92h

**Reply by m3vaz** [reply]
@jezdez This still looks to be occurring. cuda-python 11.8.1 files were uploaded 2d3h ago, but still not available in conda search

```
(base) C:\Users\mavaz>conda search -c nvidia --override-channels cuda-python=11.8.1
Loading channels: done
No match found for: cuda-python=11.8.1. Search: *cuda-python*=11.8.1

PackagesNotFoundError: The following packages are not available from current channels:

  - cuda-python=11.8.1

Current channels:

  - https://conda.anaconda.org/nvidia/win-64
  - https://conda.anaconda.org/nvidia/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.



(base) C:\Users\mavaz>conda search -c nvidia/label/main --override-channels cuda-python=11.8.1
Loading channels: done
# Name                       Version           Build  Channel
cuda-python                   11.8.1 py310h9a6739d_0  nvidia/label/main
cuda-python                   11.8.1  py37h9a6739d_0  nvidia/label/main
cuda-python                   11.8.1  py38h9a6739d_0  nvidia/label/main
cuda-python                   11.8.1  py39h9a6739d_0  nvidia/label/main
```

**Comment by barabo** [comment]
I discovered that the nvidia clone VM was stuck again, but apparently it's a different reason than before.  I was able to re-run the clone and it worked - you should see the new version of `cuda-python` now.

I'm looking into what went wrong with the clone this time.

**Comment by barabo** [comment, marked as answer]
I found the problem.  It was operator error (me).  After the last outage was over, I made an incorrect selection when restoring the nvidia clone to service.  This caused the clone VM to start, but not stop - so none of our alarms were triggered.  Unfortunately, after the VM started, the clone process did not start - so the machine was up and doing nothing.

**Reply by barabo** [reply]
It's fixed now, and I created an [issue](https://anaconda.atlassian.net/browse/CORE-1260) to add alerting for long-running clones.

**Reply by jakirkham** [reply]
Ah nice! Thanks Carl 😄

Yeah that will be helpful. Off topic, but we saw a clone in conda-forge that was taking longer than usual (~30min until CDN updated). Will this catch that sort of thing as well?

**Reply by barabo** [reply]
There's a separate alert for the conda-forge issue.  Also, the recent conda-forge issue was caused by a software upgrade to the clone that took longer than expected.
